File: mobile/lib/domain/entities/umkm_debt.dart
class UmkmDebt {
  final String id;
  final String userId;
  final String name;
  final String type; // hutang, piutang
  final double amount;
  final DateTime dueDate;
  final bool isPaid;
  final bool isSynced;
  final DateTime updatedAt;

  UmkmDebt({
    required this.id,
    required this.userId,
    required this.name,
    required this.type,
    required this.amount,
    required this.dueDate,
    this.isPaid = false,
    this.isSynced = false,
    required this.updatedAt,
  });
}

File: backend/app/Models/UmkmDebt.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Concerns\HasUuids;
use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class UmkmDebt extends Model
{
    use HasFactory, HasUuids;

    protected $fillable = [
        'user_id',
        'name',
        'type',
        'amount',
        'due_date',
        'is_paid',
    ];

    public function user()
    {
        return $this->belongsTo(User::class);
    }
}
